docs/GettingStarted/configuration/PelorusExporters.md

```diff
@@ -1,175 +1,326 @@
+### exporter_type
 
-```yaml
-exporters:
-  instances:
-  - app_name: deploytime-exporter
-    exporter_type: deploytime
-    env_from_configmaps:
-    - pelorus-config
-    - deploytime-config
-```
+- **Required:** yes
+- **Type:** string
 
-Additionally, you may want to deploy a single exporter multiple times to gather data from different sources. For example, if you wanted to pull commit data from both GitHub and a private GitHub Enterprise instance, you would deploy two instances of the Commit Time Exporter.
+    Set the exporter type. One of `deploytime`, `committime`, `failure`.
 
-Each exporter additionally takes a unique set of environment variables to further configure its integrations and behavior. These can be set by using example ConfigMap object configurations similarly to the kubernetes secrets and listing them under `env_from_configmaps` or under `env_from_secrets` accordingly. As shown below.
+### env_from_secrets
 
-```yaml
-exporters:
-  instances:
-  - app_name: committime-github
-    exporter_type: comittime
-    env_from_secrets:
-    - github-credentials
-    env_from_configmaps:
-    - pelorus-config
-    - committime-config
+- **Required:** no
+- **Type:** list
+
+    **Recommended for sensitive data**
 
-  - app_name: committime-gh-enterprise
-    exporter_type: comittime
+    List of secrets, like in the following example.
+    ```yaml
     env_from_secrets:
-    - github-enterprise-credentials
+    - example-secret
+    ```
+
+    Check the list of all available options per exporter type:
+
+    - [Deploy time](ExporterDeploytime.md#deploy-time-exporter-configuration-options)
+    - [Commit time](ExporterCommittime.md#commit-time-exporter-configuration-options)
+    - [Failure](ExporterFailure.md#failure-time-exporter-configuration-options)
+
+    **Secrets Examples**
+
+    1. To create a secret named **example-secret** in **pelorus** namespace, with the option **TOKEN** with value **token_value**, run
+        ```
+        oc create secret generic example-secret -n pelorus \
+        --from-literal=TOKEN=token_value
+        ```
+        Then, add
+        ```yaml
+        [...]
+                env_from_secrets:
+                - example-secret
+        [...]
+        ```
+        to the exporter configuration.
+
+    1. To create a secret named **other-secret** in **pelorus** namespace, with the
+
+        - option **SERVER** with value **server_url**
+        - option **API_USER** with value **username**
+        - option **TOKEN** with value **token_value**
+
+        run
+        ```
+        oc create secret generic other-secret -n pelorus \
+        --from-literal=SERVER=server_url \
+        --from-literal=API_USER=username \
+        --from-literal=TOKEN=token_value
+        ```
+        Then, add
+        ```yaml
+        [...]
+                env_from_secrets:
+                - other-secret
+        [...]
+        ```
+        to the exporter configuration.
+
+### env_from_configmaps
```

Comment:
Is there any information of the order/priority of values from secrets/configmaps and env? We have a cool way of defining one variable for all exporters and then per-exporter configs using ConfigMaps/Secrets/Env, but I am missing such use-case in this doc: 

```
    env_from_configmaps:
    - pelorus-config-for-all-exporters
    - committime-config
```

Reply:
I did not had to test priority

I will add the example

Reply:
https://github.com/konveyor/pelorus/issues/802